user/user.jsonld

```diff
@@ -1,7 +1,7 @@
 {
-    "@context": {
-        "UserProfile": "https://smartdatamodels.org/UserProfile",
-        "subcribedToAlerts": "https://vocab.egm.io/subcribedToAlerts",
-        "userInfos": "https://vocab.egm.io/userInfos"
-    }
+  "@context": {
+    "UserProfile": "https://smartdatamodels.org/UserProfile",
+    "subscribedToAlerts": "https://vocab.egm.io/subscribedToAlerts",
```

Comment:
I don't think there will be alerts (at least for now..)

Reply:
There won't be. This was a general context, we figured the UserProfile (in general) should have this attribute globally. There was just a typo I found so I updated it.

Reply:
fixing a typo here will break all the existing deployments that use this term

Reply:
don't forget this

Reply:
How would you recommend proceeding? I think for the moment it is just Conch Connect & SMBT.

Would I need to delete all the UserProfile entities for those projects? Deleting them would remove their subscriptions but the next time they login the UserProfile will be automatically created again.